### DISCUSSION TITLE
浮墨命令行工具

### DISCUSSION THREAD

**Discussion by ShroXd** [flomoapp/3rd-party-tools#16] [分享创造]
让你即使在面对终端界面时，也能抓住转瞬即逝的灵感

https://github.com/ShroXd/flomo-terminal

**Comment by plidezus** [comment]
居然还可以这样，瑞思拜

**Comment by geekdada** [comment]
这里还有一个，跨平台即装即用 https://github.com/geekdada/flomo-cli

**Reply by ShroXd** [reply]
大佬牛批！
